lib/utils.js

```diff
@@ -599,3 +599,9 @@ function getComplexSize(obj) {
   return Array.isArray(obj) ? obj.length : getComplexKeys(obj).length;
 }
 exports.getComplexSize = getComplexSize;
+
+// Returns true if a where clause is empty
+function whereIsEmpty(obj) {
```

Comment:
As this returns boolean, `isWhereEmpty`sounds more natural to me, what do you think? :)